BinaryTreeTask/BinaryTreeTask.c

```diff
@@ -0,0 +1,102 @@
+﻿#include <stdio.h>
+#include <stdlib.h>
+#include <locale.h>
+#include <stdbool.h>
+
+#include "Dictionary.h"
+#include "Test.h"
+
+int main()
+{
+    if (!test())
+    {
+        return -1;
+    }
+    setlocale(LC_ALL, "Russian");
+    int command = 0;
+    bool endOfProgram = false;
+    BinaryTree* tree = createBinaryTree();
+    while (!endOfProgram)
+    {
+        instructions();
+        scanf_s("%d", &command);
+        int key = 0;
+        switch (command)
+        {
+        case 0:
+            endOfProgram = true;
+            break;
+        case 1:
+        {
+            printf("Введите ключ: ");
+            if (scanf_s("%d", &key) != 1)
+            {
+                printf("Неверный ввод, повторите снова\n");
+                break;
+            }
+            printf("Введите значение: ");
+            char value[100];
```

Comment:
```suggestion
            char value[100] = { '\0' };
```
Неинициализированные переменные запрещает стайлгайд